pkg/products/rhsso/reconciler.go

```diff
@@ -539,7 +553,7 @@ func (r *Reconciler) exportConfig(ctx context.Context, serverClient k8sclient.Cl
 	// TODO: once the keycloak operator generates a route with a valid certificate, that
 	// should be reverted back to using the InternalURL
 	// r.Config.SetHost(kc.Status.InternalURL)
-	r.Config.SetHost(r.KeycloakHost)
+	// r.Config.SetHost(r.KeycloakHost)
```

Comment:
My guess is, this was commented out temporarily? We can remove it now, right?

Reply:
yes